src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+  line-height: 16px;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  list-style-type: none;
+  background: url(../images/star.svg) no-repeat center center;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background: url(../images/star-active.svg) no-repeat center center;
```

Comment:
The `background` shorthand property in `.stars--4 .stars__star:nth-child(-n + 4)` also includes `background-position` and `background-repeat`. You should remove the redundant `center center` from the value to follow the DRY (Don't Repeat Yourself) principle.